### ISSUE TITLE
No email notifications when card due date is reached

### ISSUE THREAD

**Issue by mstyp** [nextcloud/deck#2223]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

**Describe the bug**
No email notificatoins for when card due date is reached.

**To Reproduce**
Create a new card with a due date/time. Wait for time to be reached. check inbox.

**Expected behavior**
Recive an email when due date is reached. 
(as a feature request, it should be possible to make milestones or reminders or whatever, to nudge people over email that the due date is being approached)

**Screenshots**

**Client details:**
 - Device: [e.g. iPhone6, desktop]

<details>
<summary>Server details</summary>

**Operating system**:

**Web server:**

**Database:**

**PHP version:**

**Nextcloud version:** (see Nextcloud admin page)

**Where did you install Nextcloud from:**

**Signing status:**

```
Login as admin user into your Nextcloud and access
http://example.com/index.php/settings/integrity/failed
paste the results here.
```

**List of activated apps:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your Nextcloud installation folder
```

**Nextcloud configuration:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your Nextcloud installation folder

or

Insert your config.php content here
Make sure to remove all sensitive content such as passwords. (e.g. database password, passwordsalt, secret, smtp password, …)
```

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...

</details>

<details>
<summary>Logs</summary>

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```

#### Browser log
```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log
c) ...
```

</details>

**Comment by juliushaertl** [comment]
We currently only send out Nextcloud notifications when a due date is reached, however it might make sense to send out an activity about it so that users can opt in for an email notification.

**Comment by birkschmithuesen** [comment]
@juliushaertl But also this is not working, as described by @mstyp 
There is no email notification, when a due date is reached

**Comment by juliushaertl** [comment]
Yes, as mentioned we only issue Nextcloud notifications (in the top right notification menu), activities which might be configured by the user to receive an email about are not implemented yet, therefore the issue is still open and marked as "to develop".

**Comment by opeth281** [comment]
Hi, is there any progress - estimate finish date for this project? I found this idea very helpful for my job.

**Comment by velocity08** [comment]
Hi @juliushaertl 

haven't contributed to a project before and would like a little bit of guidance, i'm interested in assisting with this one.

Where can we get started to help with improving notifications so they are sent by email as well with additional reminders to be set before the due date.

""Cheers
G
